### PR TITLE
Add form design

### PR DESCRIPTION
**Pull request by naomiaz**

### CODE REVIEW

File: components/team-member-form/TeamMemberForm.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+            {/* FORM CONTAINER */}
+            <div className="team-member-form__form-container">
+
+              {/* FIRST COLUMN */}
+              <div className="team-member-form__first-column">
+                <img
+                  alt="Edit Avatar"
+                  className="team-member-form__avatar"
+                  src="../../static/images/avatar-naomi.jpg"
```
Comment (boris-arkenaar) [inline components/team-member-form/TeamMemberForm.jsx]: This is relative to the domain your website runs on. So you could specify it like `src="/static/images/avatar-naomi.jpg"`.

File: components/team-member-form/TeamMemberForm.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    </label>
+                  </div>
+                  <div className="team-member-form__city">
+                    <label className="team-member-form__label" htmlFor="city">
+                      City
+                      <input className="team-member-form__input" type="text" id="city" name="city" required />
+                    </label>
+                  </div>
+                </div>
+                <div className="team-member-form__socials">
```
Comment (seantijmons) [inline components/team-member-form/TeamMemberForm.jsx]: I don't see any styling for this class and also for address/bio. At the bottom of the css stylesheet you do define _first-name/_last-name/_zip/_city, do these then also need to be added here? I would leave a div out of the code if it has no functional class assigned to it.

File: components/team-member-form/team-member-form.scss [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    display: flex;
+    flex-flow: row;
+    justify-content: space-between;
+  }
+
+  &__first-name,
+  &__last-name,
+  &__zip,
+  &__city {
+    flex-basis: 49%;
```
Comment (boris-arkenaar) [inline components/team-member-form/team-member-form.scss]: Why not use a margin to separate the input boxes? That's more explicit and gives you more control over the exact width of the spacing.

File: components/team-member-overview/TeamMemberOverview.jsx [hunk 4]
```diff
@@ -15,14 +16,18 @@ class TeamMemberOverview extends React.Component {
...
-            New Humanoid
-          </button>
+          <Link href="/add-new-team-member">
+            <a className="render-whitespace--left">
+              <button
+                className="btn team-member-overview__button-new"
+                onClick={this.handleFormVisibility}
+                type="button"
+              >
+                <svg className="team-member-overview__icon-plus" />
```
Comment (boris-arkenaar) [inline components/team-member-overview/TeamMemberOverview.jsx]: Try to place the icon using css only. Making use of `background` and `::after` (with `content: ""`), removing the `svg` element from your template.

File: components/team-member-form/TeamMemberForm.jsx (hunk 2 above)
Comment (naomiaz) [reply components/team-member-form/TeamMemberForm.jsx]: Good point, removed the unused classes

File: components/team-member-form/team-member-form.scss (hunk 3 above)
Comment (naomiaz) [reply components/team-member-form/team-member-form.scss]: Fixed this

File: components/team-member-overview/TeamMemberOverview.jsx (hunk 4 above)
Comment (naomiaz) [reply components/team-member-overview/TeamMemberOverview.jsx]: Will save this for a future branch!

**Pull request merged**